schema/pgsql/upgrades/015.sql

```diff
@@ -0,0 +1,18 @@
+ALTER TABLE object
+  ADD COLUMN source_id bigint NOT NULL REFERENCES source(id) DEFAULT 1,
+  ADD COLUMN name text NOT NULL DEFAULT '',
```

Comment:
The schema migration should not leave defaults for columns set that aren't set by the full schema file. Also, if in doubt - at this stage of the project - it's better to just truncate the tables instead of leaving rows with some fake values that don't work properly (i.e. in order to migrate the existing data for this change, you'd have to recompute all IDs and change them throughout the database, otherwise, this just leaves half-broken data behind).